### PR TITLE
[API] Expose TaxonImage resource in shop

### PR DESCRIPTION
**Pull request by NoResponseMate**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.13 |
| Bug fix?        | no                                                       |
| New feature?    | yes                                                       |
| BC breaks?      | no                                                       |
| Deprecations?   | no |
| Related tickets | -                      |
| License         | MIT                                                          |

### CODE REVIEW

File: features/product/viewing_products/viewing_taxon_image.feature [hunk 1]
```diff
@@ -0,0 +1,16 @@
+@viewing_products
+Feature: Viewing a product's image on a product details page
+    In order to see images of a product
```
Comment (Wojdylak) [inline features/product/viewing_products/viewing_taxon_image.feature]:
```suggestion
    In order to see images of a taxon
```
+/- 2 lines

File: tests/Api/Shop/TaxonImagesTest.php [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Api\Shop;
```
Comment (Wojdylak) [inline tests/Api/Shop/TaxonImagesTest.php]:
```suggestion
namespace Sylius\Tests\Api\Shop;
```

File: features/product/viewing_products/viewing_taxon_image.feature [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    As a Visitor
+    I want to be able to view an image of a single product
```
Comment (GSadee) [inline features/product/viewing_products/viewing_taxon_image.feature]:
```suggestion
Feature: Viewing a taxon's image on a taxon details page
    In order to see images of a taxon
    As a Visitor
    I want to be able to view an image of a single taxon
```

File: tests/Api/Shop/TaxonImagesTest.php [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+use Symfony\Component\HttpFoundation\Response;
+
+final class TaxonImagesTest extends JsonApiTestCase
+{
+    /** @test */
+    public function it_gets_a_taxon_image(): void
+    {
+        $fixtures = $this->loadFixturesFromFiles([
+            'taxon_image.yaml',
+        ]);
```
Comment (GSadee) [inline tests/Api/Shop/TaxonImagesTest.php]:
```suggestion
        $fixtures = $this->loadFixturesFromFile('taxon_image.yaml');
```

File: tests/Api/Shop/TaxonsTest.php [hunk 5]
```diff
@@ -19,33 +19,45 @@
...
-        $this->client->request(method: 'GET', uri: '/api/v2/shop/taxons', server: self::CONTENT_TYPE_HEADER);
+        $this->client->request(
+            method: 'GET',
+            uri: '/api/v2/shop/taxons',
+            server: self::CONTENT_TYPE_HEADER,
+        );
         $response = $this->client->getResponse();
 
-        $this->assertResponse($response, 'shop/taxon/get_taxonomy_response', Response::HTTP_OK);
+        $this->assertResponse($response, 'shop/taxon/get_taxons', Response::HTTP_OK);
```
Comment (GSadee) [inline tests/Api/Shop/TaxonsTest.php:33]: You are not consistent within this PR in the context of using the suffix `_response`

**Review by jakubtobiasz — APPROVED** [review]

**Comment by jakubtobiasz** [comment]
Thank you, @NoResponseMate!

**Pull request merged**